.drone.yml

```diff
@@ -2,6 +2,10 @@ workspace:
   base: /go
 
 pipeline:
+  clone:
+    image: plugins/git
+    tags: true
```

Comment:
This is example of gorush repo: https://github.com/appleboy/gorush/blob/master/.drone.yml#L44-L50

Drone Log: http://drone.wu-boy.com/appleboy/gorush/267/1

You can see git clone all tag if set `tags` as `true`.